src/plugins/vis_type_timeseries/public/application/components/aggs/percentile_rank/percentile_rank.tsx

```diff
@@ -52,7 +69,7 @@ export const PercentileRankAgg = (props) => {
   const handleChange = createChangeHandler(props.onChange, model);
   const handleSelectChange = createSelectHandler(handleChange);
 
-  const handlePercentileRankValuesChange = (values) => {
+  const handlePercentileRankValuesChange = (values: Array<string | null> | null | undefined) => {
     handleChange(
       assign({}, model, {
         values,
```

Comment:
```suggestion
  const handlePercentileRankValuesChange = (values: MetricsItemsSchema['values']) => {
    handleChange(
      assign({}, model, {
        values,
```

You could also get rid of lodash here (we are trying to do it if possible):
```
handleChange({
  ...model,
  values
})